pkgs/development/compilers/silice/default.nix

```diff
@@ -78,10 +95,16 @@ stdenv.mkDerivation rec {
       pipeline_sort = testProject "pipeline_sort";
     };
 
-  meta = with lib; {
+  passthru.updateScript = unstableGitUpdater { };
```

Comment:
Does this updater still output versions prefixed with "unstable"? If so maybe we should consider not prefixing "0-"

Reply:
No.